Error/ErrorHandler.php

```diff
@@ -129,4 +133,32 @@ public function handleErrors(ExecutionResult $executionResult, $throwRawExceptio
             $executionResult->extensions['warnings'] = array_map(['GraphQL\Error', 'formatError'], $exceptions['extensions']['warnings']);
         }
     }
+
+    /**
+     * Tries to convert a raw exception into a user warning or error
+     * that is displayed to the user.
+     *
+     * @param \Exception $rawException
+     * @return \Exception
+     */
+    protected function convertException(\Exception $rawException = null)
+    {
+        if (empty($rawException)) {
```

Comment:
This test could be rewritten to make it easier to understand. Maybe something like:

```php
if (null === $rawException) {
    return null;
}
```